data/i18n/en/en.toml

```
@@ -202,6 +202,14 @@ other = "Objectives"
 [options_heading]
 other = "Options"
 
+[outdated_page_message]
+other = "Kubernetes authors consider this page to be outdated because REASON. We suggest that you use more recent resources."
+[outdated_page_reason]
+other = "it is more than a year old"
+
+[outdated_page_title]
+other = "Outdated page"
```

```suggestion
other = "Outdated article"
```
(I'd also change the name to, eg,`deprecation_warning_blog_over_one_year_old_title`)